### PR TITLE
Update core-retrieval to use atomic siva copy

### PR DESCRIPTION
**Pull request by jfontan**
Related to https://github.com/src-d/core-retrieval/pull/44

### CODE REVIEW

File: glide.lock [hunk 1]
```diff
@@ -42,7 +42,7 @@ imports:
 - name: github.com/lann/ps
   version: 62de8c46ede02a7675c4c79c84883eb164cb71e3
 - name: github.com/lib/pq
-  version: 8c6ee72f3e6bcb1542298dd5f76cb74af9742cec
+  version: 83612a56d3dd153a94a629cd64925371c9adad78
```
Comment (ajnavarro) [inline glide.lock]: All these packages updates are necessary?

Comment (jfontan) [reply glide.lock]: No, this was automatically updated by `glide up`.

Comment (ajnavarro) [reply glide.lock]: Then, just to avoid library incompatibility problems, can you only update the core-retrieval dependency?

**Comment by jfontan** [comment]
I'm closing this PR and will create a new one when temporary dir problems are merged.

Related: https://github.com/src-d/borges/issues/190

**Pull request closed without merging**